AnalysisTools/plugins/PATObjectCounter.cc

```diff
@@ -43,32 +43,36 @@ class PATObjectCounter : public edm::stream::EDProducer<>
   virtual void produce(edm::Event& iEvent, const edm::EventSetup& iSetup);
 
   const edm::EDGetTokenT<edm::View<T> > srcToken_;
-  const std::vector<std::string> cuts_;
+  const std::vector<std::string> cut_strings_;
   const std::vector<std::string> labels_;
+  std::vector<StringCutObjectSelector<T>> cuts_;
```

Comment:
It may not ever matter, but we might want to use `StringCutObjectSelector<T,true>`, just for robustness.